src/components/Navbar/nav-style.css

```diff
@@ -6,8 +6,42 @@
     transition: background-size 300ms ease-in;
 }
 
+.login:hover{
+    background-color: black;
+    color: white;
+    transition: all ease-in 0.4s;
+}
+
+/* @media (top:10vh) {
+
+    .navbar{
+        background-color: black;
+    }
+} */
+
+.navbar.active {
+    background-color: rgb(0, 0, 0);
+
+}
+
+.navbar.active a{
+    transition: all ease 0.3s;
+    color:white;
+}
+
+.navbar.active button{
+    transition: all ease-in 0.3s;
+
+    color: white;
+}
+
+.login:hover{
+    border: 0.5px white solid;
+}
+
 .nav-btn:hover
 {
     transform: scale(1.15);
     background-size: 1100% ;
-}
\ No newline at end of file
+} 
```

Comment:
Kindly choose a color which would match the existing color scheme of the application

Reply:
okk sure

Reply:
@geekymeeky  I find this color matching the theme of our website
![Screenshot 2022-03-17 172345](https://user-images.githubusercontent.com/39373956/158803422-cefe469e-c91e-47be-aa74-f0521eb5f8fb.jpg)